src/components/Datepicker/Datepicker.tsx

```diff
@@ -97,36 +99,39 @@ export const Datepicker: FC<DatepickerProps> = ({
   labelClearButton = 'Clear',
   showTodayButton = true,
   labelTodayButton = 'Today',
-  defaultDate = new Date(),
+  defaultValue = new Date(),
   minDate,
   maxDate,
   language = 'en',
   weekStart = WeekStart.Sunday,
   className,
   theme: customTheme = {},
   onSelectedDateChanged,
+  value,
+  label = 'No date selected',
```

Comment:
This should be "Select date", see https://flowbite.com/docs/plugins/datepicker/#datepicker-example

We are aiming to keep the React library docs page for the component as close to identical to the upstream Flowbite library (and docs on flowbite.com) as possible.